app/schemas/schemas.coffee

```diff
@@ -125,6 +125,13 @@ me.extendSearchableProperties = (schema) ->
   schema.properties = {} unless schema.properties?
   _.extend(schema.properties, searchableProps())
 
+algoliaSearchableProps = ->
+  _algoliaObjectID: { type: 'string' }
```

Comment:
Field is optional, is `required: false` the default?

Reply:
It should be the default. We only seem to opt into required.